diff -Naupr ogmrip-0.11.2.orig/configure ogmrip-0.11.2/configure
--- ogmrip-0.11.2.orig/configure	2007-12-22 15:42:44.000000000 +0100
+++ ogmrip-0.11.2/configure	2008-05-11 23:28:32.000000000 +0200
@@ -21772,7 +21772,7 @@ cat >>conftest.$ac_ext <<_ACEOF
 #include <dvdread/ifo_types.h>
 #include <dvdread/ifo_read.h>
 #include <dvdread/nav_read.h>
-int main(void) { if (DVDREAD_VERSION < 907) return 1; return 0; }
+int main(void) { if (DVDREAD_VERSION < 904) return 1; return 0; }
 
 _ACEOF
 rm -f conftest$ac_exeext
diff -Naupr ogmrip-0.11.2.orig/configure.in ogmrip-0.11.2/configure.in
--- ogmrip-0.11.2.orig/configure.in	2008-05-11 23:28:59.000000000 +0200
+++ ogmrip-0.11.2/configure.in	2007-11-05 20:52:57.000000000 +0100
@@ -180,7 +180,7 @@ AC_TRY_RUN([
 #include <dvdread/ifo_types.h>
 #include <dvdread/ifo_read.h>
 #include <dvdread/nav_read.h>
-int main(void) { if (DVDREAD_VERSION < 907) return 1; return 0; }
+int main(void) { if (DVDREAD_VERSION < 904) return 1; return 0; }
 ],,AC_MSG_ERROR(Unable to find libdvdread >= 0.9.7))
 
 dnl **************************************************************
