Index: libogmrip/ogmrip-fs.c
===================================================================
--- libogmrip/ogmrip-fs.c	(revision 2063)
+++ libogmrip/ogmrip-fs.c	(working copy)
@@ -51,7 +51,7 @@
  * 
  * Returns: The temporaty directory
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_fs_get_tmp_dir (void)
 {
   if (!ogmrip_tmp_dir)
@@ -618,7 +618,7 @@
  * 
  * Returns: The extension, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_fs_get_extension (const gchar *filename)
 {
   gchar *dot;
Index: libogmrip/ogmrip-chapters.c
===================================================================
--- libogmrip/ogmrip-chapters.c	(revision 2063)
+++ libogmrip/ogmrip-chapters.c	(working copy)
@@ -205,7 +205,7 @@
  *
  * Returns: The label
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_chapters_get_label (OGMRipChapters *chapters, guint n)
 {
   g_return_val_if_fail (OGMRIP_IS_CHAPTERS (chapters), NULL);
Index: libogmrip/ogmrip-fs.h
===================================================================
--- libogmrip/ogmrip-fs.h	(revision 2063)
+++ libogmrip/ogmrip-fs.h	(working copy)
@@ -30,7 +30,7 @@
 #define g_chdir chdir
 #endif
 
-G_CONST_RETURN
+const
 gchar *  ogmrip_fs_get_tmp_dir      (void);
 void     ogmrip_fs_set_tmp_dir      (const gchar  *dir);
 
@@ -71,7 +71,7 @@
 
 gchar *  ogmrip_fs_get_full_path    (const gchar  *filename);
 
-G_CONST_RETURN
+const
 gchar *  ogmrip_fs_get_extension    (const gchar  *filename);
 gchar *  ogmrip_fs_set_extension    (const gchar  *filename,
                                      const gchar  *extension);
Index: libogmrip/ogmrip-codec.c
===================================================================
--- libogmrip/ogmrip-codec.c	(revision 2063)
+++ libogmrip/ogmrip-codec.c	(working copy)
@@ -294,7 +294,7 @@
  *
  * Returns: the filename, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_codec_get_output (OGMRipCodec *codec)
 {
   g_return_val_if_fail (OGMRIP_IS_CODEC (codec), NULL);
Index: libogmrip/ogmrip-chapters.h
===================================================================
--- libogmrip/ogmrip-chapters.h	(revision 2063)
+++ libogmrip/ogmrip-chapters.h	(working copy)
@@ -48,7 +48,7 @@
 GType         ogmrip_chapters_get_type  (void);
 OGMJobSpawn * ogmrip_chapters_new       (OGMDvdTitle    *title,
                                          const gchar    *output);
-G_CONST_RETURN 
+const 
 gchar *       ogmrip_chapters_get_label (OGMRipChapters *chapters, 
                                          guint          n);
 void          ogmrip_chapters_set_label (OGMRipChapters *chapters, 
Index: libogmrip/ogmrip-codec.h
===================================================================
--- libogmrip/ogmrip-codec.h	(revision 2063)
+++ libogmrip/ogmrip-codec.h	(working copy)
@@ -53,7 +53,7 @@
 };
 
 GType         ogmrip_codec_get_type            (void);
-G_CONST_RETURN
+const
 gchar *       ogmrip_codec_get_output          (OGMRipCodec *codec);
 void          ogmrip_codec_set_output          (OGMRipCodec *codec,
                                                 const gchar *output);
Index: libogmrip/ogmrip-lavc.c
===================================================================
--- libogmrip/ogmrip-lavc.c	(revision 2063)
+++ libogmrip/ogmrip-lavc.c	(working copy)
@@ -135,7 +135,7 @@
   return CLAMP (quantizer, 2, 31);
 }
 
-static G_CONST_RETURN gchar *
+static const gchar *
 ogmrip_lavc_get_codec (OGMRipLavc *lavc)
 {
   OGMRipLavcClass *klass;
Index: libogmrip/ogmrip-lavc.h
===================================================================
--- libogmrip/ogmrip-lavc.h	(revision 2063)
+++ libogmrip/ogmrip-lavc.h	(working copy)
@@ -69,7 +69,7 @@
   OGMRipVideoCodecClass parent_class;
 
   /* vtable */
-  G_CONST_RETURN gchar * (* get_codec) (void);
+  const gchar * (* get_codec) (void);
 };
 
 /**
Index: libogmrip/ogmrip-encoding.c
===================================================================
--- libogmrip/ogmrip-encoding.c	(revision 2063)
+++ libogmrip/ogmrip-encoding.c	(working copy)
@@ -359,7 +359,7 @@
   return data1->nr - data2->nr;
 }
 
-static G_CONST_RETURN gchar *
+static const gchar *
 ogmrip_encoding_get_device (OGMRipEncoding *encoding)
 {
   g_return_val_if_fail (OGMRIP_IS_ENCODING (encoding), NULL);
@@ -3550,7 +3550,7 @@
  *
  * Returns: The id, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_encoding_get_id (OGMRipEncoding *encoding)
 {
   g_return_val_if_fail (OGMRIP_IS_ENCODING (encoding), NULL);
@@ -3582,7 +3582,7 @@
  *
  * Returns: The profile' section, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_encoding_get_profile (OGMRipEncoding *encoding)
 {
   g_return_val_if_fail (OGMRIP_IS_ENCODING (encoding), NULL);
@@ -3622,7 +3622,7 @@
  *
  * Returns: The name of the encoding, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_encoding_get_label (OGMRipEncoding *encoding)
 {
   g_return_val_if_fail (OGMRIP_IS_ENCODING (encoding), NULL);
@@ -3669,7 +3669,7 @@
  *
  * Returns: The name of the chapter, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_encoding_get_chapter_label (OGMRipEncoding *encoding, guint nr)
 {
   OGMRipChapterData *data;
@@ -3829,7 +3829,7 @@
  *
  * Returns: The output filename, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_encoding_get_filename (OGMRipEncoding *encoding)
 {
   g_return_val_if_fail (OGMRIP_IS_ENCODING (encoding), NULL);
@@ -3868,7 +3868,7 @@
  *
  * Returns: The log filename, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_encoding_get_logfile (OGMRipEncoding *encoding)
 {
   g_return_val_if_fail (OGMRIP_IS_ENCODING (encoding), NULL);
@@ -4007,7 +4007,7 @@
  *
  * Returns: The FourCC, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_encoding_get_fourcc (OGMRipEncoding *encoding)
 {
   g_return_val_if_fail (OGMRIP_IS_ENCODING (encoding), NULL);
Index: libogmrip/ogmrip-encoding.h
===================================================================
--- libogmrip/ogmrip-encoding.h	(revision 2063)
+++ libogmrip/ogmrip-encoding.h	(working copy)
@@ -362,16 +362,16 @@
 
 guint32             ogmrip_encoding_get_flags              (OGMRipEncoding          *encoding);
 
-G_CONST_RETURN
+const
 gchar *             ogmrip_encoding_get_id                 (OGMRipEncoding          *encoding);
 OGMDvdTitle *       ogmrip_encoding_get_title              (OGMRipEncoding          *encoding);
 
-G_CONST_RETURN
+const
 gchar *             ogmrip_encoding_get_profile            (OGMRipEncoding          *encoding);
 void                ogmrip_encoding_set_profile            (OGMRipEncoding          *encoding,
                                                             const gchar             *profile);
 
-G_CONST_RETURN
+const
 gchar *             ogmrip_encoding_get_label              (OGMRipEncoding          *encoding);
 void                ogmrip_encoding_set_label              (OGMRipEncoding          *encoding,
                                                             const gchar             *label);
@@ -386,7 +386,7 @@
 gint                ogmrip_encoding_get_chapters_language  (OGMRipEncoding          *encoding);
 void                ogmrip_encoding_set_chapters_language  (OGMRipEncoding          *encoding,
                                                             guint                   language);
-G_CONST_RETURN
+const
 gchar *             ogmrip_encoding_get_chapter_label      (OGMRipEncoding          *encoding,
                                                             guint                   nr);
 void                ogmrip_encoding_set_chapter_label      (OGMRipEncoding          *encoding,
@@ -404,11 +404,11 @@
 gboolean            ogmrip_encoding_get_keep_tmp_files     (OGMRipEncoding          *encoding);
 void                ogmrip_encoding_set_keep_tmp_files     (OGMRipEncoding          *encoding,
                                                             gboolean                keep_tmp_files);
-G_CONST_RETURN
+const
 gchar *             ogmrip_encoding_get_filename           (OGMRipEncoding          *encoding);
 void                ogmrip_encoding_set_filename           (OGMRipEncoding          *encoding,
                                                             const gchar             *filename);
-G_CONST_RETURN
+const
 gchar *             ogmrip_encoding_get_logfile            (OGMRipEncoding          *encoding);
 
 gint                ogmrip_encoding_get_threads            (OGMRipEncoding          *encoding);
@@ -423,7 +423,7 @@
 gboolean            ogmrip_encoding_set_container_type     (OGMRipEncoding          *encoding,
                                                             GType                   type,
                                                             GError                  **error);
-G_CONST_RETURN
+const
 gchar *             ogmrip_encoding_get_fourcc             (OGMRipEncoding          *encoding);
 void                ogmrip_encoding_set_fourcc             (OGMRipEncoding          *encoding,
                                                             const gchar             *fourcc);
Index: libogmrip/ogmrip-plugin.c
===================================================================
--- libogmrip/ogmrip-plugin.c	(revision 2063)
+++ libogmrip/ogmrip-plugin.c	(working copy)
@@ -367,7 +367,7 @@
   return NULL;
 }
 
-static G_CONST_RETURN gchar *
+static const gchar *
 ogmrip_plugin_get_codec_name (GSList *list, GType type)
 {
   OGMRipPlugin *plugin;
@@ -542,7 +542,7 @@
  *
  * Returns: The name of the container, or NULL 
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_plugin_get_container_name (GType container)
 {
   OGMRipContainerPlugin *plugin;
@@ -740,7 +740,7 @@
  *
  * Returns: The name of the video codec, or NULL 
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_plugin_get_video_codec_name (GType codec)
 {
   g_return_val_if_fail (g_type_is_a (codec, OGMRIP_TYPE_VIDEO_CODEC), NULL);
@@ -926,7 +926,7 @@
  *
  * Returns: The name of the audio codec, or NULL 
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_plugin_get_audio_codec_name (GType codec)
 {
   g_return_val_if_fail (g_type_is_a (codec, OGMRIP_TYPE_AUDIO_CODEC), NULL);
@@ -1070,7 +1070,7 @@
  *
  * Returns: The name of the subtitle codec, or NULL 
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_plugin_get_subp_codec_name (GType codec)
 {
   g_return_val_if_fail (g_type_is_a (codec, OGMRIP_TYPE_SUBP_CODEC), NULL);
Index: libogmrip/ogmrip-plugin.h
===================================================================
--- libogmrip/ogmrip-plugin.h	(revision 2063)
+++ libogmrip/ogmrip-plugin.h	(working copy)
@@ -196,7 +196,7 @@
 GType     ogmrip_plugin_get_nth_container       (guint n);
 GType     ogmrip_plugin_get_container_by_name   (const gchar *name);
 gint      ogmrip_plugin_get_container_index     (GType container);
-G_CONST_RETURN
+const
 gchar *   ogmrip_plugin_get_container_name      (GType container);
 gboolean  ogmrip_plugin_get_container_bframes   (GType container);
 gint      ogmrip_plugin_get_container_max_audio (GType container);
@@ -216,7 +216,7 @@
 GType     ogmrip_plugin_get_nth_video_codec     (guint n);
 GType     ogmrip_plugin_get_video_codec_by_name (const gchar *name);
 gint      ogmrip_plugin_get_video_codec_index   (GType codec);
-G_CONST_RETURN
+const
 gchar *   ogmrip_plugin_get_video_codec_name    (GType codec);
 gint      ogmrip_plugin_get_video_codec_format  (GType codec);
 gint      ogmrip_plugin_get_video_codec_passes  (GType codec);
@@ -236,7 +236,7 @@
 GType     ogmrip_plugin_get_nth_audio_codec     (guint n);
 GType     ogmrip_plugin_get_audio_codec_by_name (const gchar *name);
 gint      ogmrip_plugin_get_audio_codec_index   (GType codec);
-G_CONST_RETURN
+const
 gchar *   ogmrip_plugin_get_audio_codec_name    (GType codec);
 gint      ogmrip_plugin_get_audio_codec_format  (GType codec);
 GModule * ogmrip_plugin_get_audio_codec_module  (GType codec);
@@ -254,7 +254,7 @@
 GType     ogmrip_plugin_get_nth_subp_codec     (guint n);
 GType     ogmrip_plugin_get_subp_codec_by_name (const gchar *name);
 gint      ogmrip_plugin_get_subp_codec_index   (GType codec);
-G_CONST_RETURN
+const
 gchar *   ogmrip_plugin_get_subp_codec_name    (GType codec);
 gint      ogmrip_plugin_get_subp_codec_format  (GType codec);
 gboolean  ogmrip_plugin_get_subp_codec_text    (GType codec);
Index: libogmrip/ogmrip-edl.c
===================================================================
--- libogmrip/ogmrip-edl.c	(revision 2063)
+++ libogmrip/ogmrip-edl.c	(working copy)
@@ -217,7 +217,7 @@
  *
  * Returns: The filename
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_edl_get_filename (OGMRipEdl *edl)
 {
   g_return_val_if_fail (edl != NULL, NULL);
Index: libogmrip/ogmrip-edl.h
===================================================================
--- libogmrip/ogmrip-edl.h	(revision 2063)
+++ libogmrip/ogmrip-edl.h	(working copy)
@@ -56,7 +56,7 @@
 void        ogmrip_edl_ref             (OGMRipEdl       *edl);
 void        ogmrip_edl_unref           (OGMRipEdl       *edl);
 
-G_CONST_RETURN
+const
 gchar *     ogmrip_edl_get_filename    (OGMRipEdl       *edl);
 
 void        ogmrip_edl_add             (OGMRipEdl       *edl,
Index: libogmrip/ogmrip-subp-codec.c
===================================================================
--- libogmrip/ogmrip-subp-codec.c	(revision 2063)
+++ libogmrip/ogmrip-subp-codec.c	(working copy)
@@ -347,7 +347,7 @@
  *
  * Returns: the track name
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_subp_codec_get_label (OGMRipSubpCodec *subp)
 {
   g_return_val_if_fail (OGMRIP_IS_SUBP_CODEC (subp), NULL);
Index: libogmrip/ogmrip-subp-codec.h
===================================================================
--- libogmrip/ogmrip-subp-codec.h	(revision 2063)
+++ libogmrip/ogmrip-subp-codec.h	(working copy)
@@ -62,7 +62,7 @@
 gint               ogmrip_subp_codec_get_newline         (OGMRipSubpCodec  *subp);
 void               ogmrip_subp_codec_set_label           (OGMRipSubpCodec  *subp,
                                                           const gchar      *label);
-G_CONST_RETURN
+const
 gchar *            ogmrip_subp_codec_get_label           (OGMRipSubpCodec  *subp);
 
 G_END_DECLS
Index: libogmrip/ogmrip-audio-codec.c
===================================================================
--- libogmrip/ogmrip-audio-codec.c	(revision 2063)
+++ libogmrip/ogmrip-audio-codec.c	(working copy)
@@ -465,7 +465,7 @@
  *
  * Returns: the track name
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_audio_codec_get_label (OGMRipAudioCodec *audio)
 {
   g_return_val_if_fail (OGMRIP_IS_AUDIO_CODEC (audio), NULL);
Index: libogmrip/ogmrip-audio-codec.h
===================================================================
--- libogmrip/ogmrip-audio-codec.h	(revision 2063)
+++ libogmrip/ogmrip-audio-codec.h	(working copy)
@@ -71,7 +71,7 @@
 gint                ogmrip_audio_codec_get_sample_rate       (OGMRipAudioCodec    *audio);
 void                ogmrip_audio_codec_set_label             (OGMRipAudioCodec    *audio,
                                                               const gchar         *label);
-G_CONST_RETURN
+const
 gchar *             ogmrip_audio_codec_get_label             (OGMRipAudioCodec    *audio);
 gint                ogmrip_audio_codec_get_samples_per_frame (OGMRipAudioCodec    *audio);
 
Index: libogmrip/ogmrip-container.c
===================================================================
--- libogmrip/ogmrip-container.c	(revision 2063)
+++ libogmrip/ogmrip-container.c	(working copy)
@@ -325,7 +325,7 @@
  *
  * Returns: The filename, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_container_get_output (OGMRipContainer *container)
 {
   g_return_val_if_fail (OGMRIP_IS_CONTAINER (container), NULL);
@@ -358,7 +358,7 @@
  *
  * Returns: The label, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_container_get_label (OGMRipContainer *container)
 {
   g_return_val_if_fail (OGMRIP_IS_CONTAINER (container), NULL);
@@ -390,7 +390,7 @@
  *
  * Returns: The FourCC, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmrip_container_get_fourcc (OGMRipContainer *container)
 {
   g_return_val_if_fail (OGMRIP_IS_CONTAINER (container), NULL);
Index: libogmrip/ogmrip-container.h
===================================================================
--- libogmrip/ogmrip-container.h	(revision 2063)
+++ libogmrip/ogmrip-container.h	(working copy)
@@ -91,16 +91,16 @@
 void               ogmrip_container_set_options          (OGMRipContainer          *container,
                                                           const gchar              *section);
 
-G_CONST_RETURN
+const
 gchar *            ogmrip_container_get_output           (OGMRipContainer          *container);
 void               ogmrip_container_set_output           (OGMRipContainer          *container,
                                                           const gchar              *output);
-G_CONST_RETURN
+const
 gchar *            ogmrip_container_get_label            (OGMRipContainer          *container);
 void               ogmrip_container_set_label            (OGMRipContainer          *container,
                                                           const gchar              *label);
 
-G_CONST_RETURN
+const
 gchar *            ogmrip_container_get_fourcc           (OGMRipContainer          *container);
 void               ogmrip_container_set_fourcc           (OGMRipContainer          *container,
                                                           const gchar              *fourcc);
Index: libogmdvd/ogmdvd-labels.c
===================================================================
--- libogmdvd/ogmdvd-labels.c	(revision 2063)
+++ libogmdvd/ogmdvd-labels.c	(working copy)
@@ -248,7 +248,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_video_format_label (gint format)
 {
   static const gchar *video_format[] = 
@@ -272,7 +272,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_display_aspect_label (gint aspect)
 {
   static gchar *display_aspect[] =
@@ -296,7 +296,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_audio_format_label (gint format)
 {
   static gchar *audio_format[] =
@@ -324,7 +324,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_audio_channels_label (gint channels)
 {
   static gchar *audio_channels[] =
@@ -352,7 +352,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_audio_quantization_label (gint quantization)
 {
   static gchar *audio_quantization[] = 
@@ -376,7 +376,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_audio_content_label (gint content)
 {
   static gchar *audio_content[] = 
@@ -401,7 +401,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_subp_content_label (gint content)
 {
   static gchar *subp_content[] = 
@@ -437,7 +437,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_language_label (gint code)
 {
   const gchar *lang;
@@ -460,7 +460,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_language_iso639_1 (gint code)
 {
   static gchar lang[3];
@@ -489,7 +489,7 @@
  *
  * Returns: A constant string, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_get_language_iso639_2 (gint code)
 {
   const gchar *lang;
Index: libogmdvd/ogmdvd-labels.h
===================================================================
--- libogmdvd/ogmdvd-labels.h	(revision 2063)
+++ libogmdvd/ogmdvd-labels.h	(working copy)
@@ -23,16 +23,16 @@
 
 G_BEGIN_DECLS
 
-G_CONST_RETURN gchar * ogmdvd_get_video_format_label       (gint format);
-G_CONST_RETURN gchar * ogmdvd_get_display_aspect_label     (gint aspect);
-G_CONST_RETURN gchar * ogmdvd_get_audio_format_label       (gint format);
-G_CONST_RETURN gchar * ogmdvd_get_audio_channels_label     (gint channels);
-G_CONST_RETURN gchar * ogmdvd_get_audio_quantization_label (gint quantization);
-G_CONST_RETURN gchar * ogmdvd_get_audio_content_label      (gint content);
-G_CONST_RETURN gchar * ogmdvd_get_subp_content_label       (gint content);
-G_CONST_RETURN gchar * ogmdvd_get_language_label           (gint code);
-G_CONST_RETURN gchar * ogmdvd_get_language_iso639_1        (gint code);
-G_CONST_RETURN gchar * ogmdvd_get_language_iso639_2        (gint code);
+const gchar * ogmdvd_get_video_format_label       (gint format);
+const gchar * ogmdvd_get_display_aspect_label     (gint aspect);
+const gchar * ogmdvd_get_audio_format_label       (gint format);
+const gchar * ogmdvd_get_audio_channels_label     (gint channels);
+const gchar * ogmdvd_get_audio_quantization_label (gint quantization);
+const gchar * ogmdvd_get_audio_content_label      (gint content);
+const gchar * ogmdvd_get_subp_content_label       (gint content);
+const gchar * ogmdvd_get_language_label           (gint code);
+const gchar * ogmdvd_get_language_iso639_1        (gint code);
+const gchar * ogmdvd_get_language_iso639_2        (gint code);
 
 G_END_DECLS
 
Index: libogmdvd/ogmdvd-title.c
===================================================================
--- libogmdvd/ogmdvd-title.c	(revision 2063)
+++ libogmdvd/ogmdvd-title.c	(working copy)
@@ -437,7 +437,7 @@
  *
  * Returns: a constant array of 16 integers, or NULL
  */
-G_CONST_RETURN guint *
+const guint *
 ogmdvd_title_get_palette (OGMDvdTitle *title)
 {
   g_return_val_if_fail (title != NULL, NULL);
Index: libogmdvd/ogmdvd-disc.c
===================================================================
--- libogmdvd/ogmdvd-disc.c	(revision 2063)
+++ libogmdvd/ogmdvd-disc.c	(working copy)
@@ -134,7 +134,7 @@
   return reader;
 }
 
-static G_CONST_RETURN gchar *
+static const gchar *
 dvd_reader_get_id (dvd_reader_t *reader)
 {
   static gchar str[33];
@@ -788,7 +788,7 @@
  *
  * Returns: The label of the DVD, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_disc_get_label (OGMDvdDisc *disc)
 {
   g_return_val_if_fail (disc != NULL, NULL);
@@ -804,7 +804,7 @@
  *
  * Returns: The identifier, or NULL
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_disc_get_id (OGMDvdDisc *disc)
 {
   g_return_val_if_fail (disc != NULL, NULL);
@@ -820,7 +820,7 @@
  *
  * Returns: The device of the DVD.
  */
-G_CONST_RETURN gchar *
+const gchar *
 ogmdvd_disc_get_device (OGMDvdDisc *disc)
 {
   g_return_val_if_fail (disc != NULL, NULL);
Index: libogmdvd/ogmdvd-title.h
===================================================================
--- libogmdvd/ogmdvd-title.h	(revision 2063)
+++ libogmdvd/ogmdvd-title.h	(working copy)
@@ -54,7 +54,7 @@
 void                ogmdvd_title_get_aspect_ratio     (OGMDvdTitle  *title,
                                                        guint        *numerator,
                                                        guint        *denominator);
-G_CONST_RETURN
+const
 guint *             ogmdvd_title_get_palette          (OGMDvdTitle  *title);
 gint                ogmdvd_title_get_n_angles         (OGMDvdTitle  *title);
 gint                ogmdvd_title_get_n_chapters       (OGMDvdTitle  *title);
Index: libogmdvd/ogmdvd-disc.h
===================================================================
--- libogmdvd/ogmdvd-disc.h	(revision 2063)
+++ libogmdvd/ogmdvd-disc.h	(working copy)
@@ -67,11 +67,11 @@
 
 void          ogmdvd_disc_ref           (OGMDvdDisc  *disc);
 void          ogmdvd_disc_unref         (OGMDvdDisc  *disc);
-G_CONST_RETURN
+const
 gchar *       ogmdvd_disc_get_label     (OGMDvdDisc  *disc);
-G_CONST_RETURN
+const
 gchar *       ogmdvd_disc_get_id        (OGMDvdDisc  *disc);
-G_CONST_RETURN
+const
 gchar *       ogmdvd_disc_get_device    (OGMDvdDisc  *disc);
 gint64        ogmdvd_disc_get_vmg_size  (OGMDvdDisc  *disc);
 gint          ogmdvd_disc_get_n_titles  (OGMDvdDisc  *disc);
