diff --git a/libogmdvd-gtk/ogmdvd-chapter-list.h b/libogmdvd-gtk/ogmdvd-chapter-list.h
index c9c9bf8..4543285 100644
--- a/libogmdvd-gtk/ogmdvd-chapter-list.h
+++ b/libogmdvd-gtk/ogmdvd-chapter-list.h
@@ -21,7 +21,7 @@
 
 #include <ogmdvd.h>
 
-#include <gtk/gtkcombobox.h>
+#include <gtk/gtk.h>
 
 G_BEGIN_DECLS
 
diff --git a/libogmdvd-gtk/ogmdvd-drive-chooser-widget.h b/libogmdvd-gtk/ogmdvd-drive-chooser-widget.h
index d6d5b8e..be0ad89 100644
--- a/libogmdvd-gtk/ogmdvd-drive-chooser-widget.h
+++ b/libogmdvd-gtk/ogmdvd-drive-chooser-widget.h
@@ -20,7 +20,7 @@
 #define __OGMDVD_DRIVE_CHOOSER_WIDGET_H__
 
 #include <ogmdvd-drive-chooser.h>
-#include <gtk/gtkcombobox.h>
+#include <gtk/gtk.h>
 
 G_BEGIN_DECLS
 
diff --git a/libogmdvd-gtk/ogmdvd-title-chooser-widget.h b/libogmdvd-gtk/ogmdvd-title-chooser-widget.h
index f0045f1..1fc6a52 100644
--- a/libogmdvd-gtk/ogmdvd-title-chooser-widget.h
+++ b/libogmdvd-gtk/ogmdvd-title-chooser-widget.h
@@ -20,7 +20,7 @@
 #define __OGMDVD_TITLE_CHOOSER_WIDGET_WIDGET_H__
 
 #include <ogmdvd-title-chooser.h>
-#include <gtk/gtkcombobox.h>
+#include <gtk/gtk.h>
 
 G_BEGIN_DECLS
 
diff --git a/libogmrip-gtk/ogmrip-source-chooser-widget.h b/libogmrip-gtk/ogmrip-source-chooser-widget.h
index ad25528..52f92ee 100644
--- a/libogmrip-gtk/ogmrip-source-chooser-widget.h
+++ b/libogmrip-gtk/ogmrip-source-chooser-widget.h
@@ -20,7 +20,7 @@
 #define __OGMRIP_SOURCE_CHOOSER_WIDGET_H__
 
 #include <ogmrip-source-chooser.h>
-#include <gtk/gtkcombobox.h>
+#include <gtk/gtk.h>
 
 G_BEGIN_DECLS
 
