diff -u -r ogmrip-0.13.6.orig/libogmrip-gtk/ogmrip-x264-options.c ogmrip-0.13.6/libogmrip-gtk/ogmrip-x264-options.c
--- ogmrip-0.13.6.orig/libogmrip-gtk/ogmrip-x264-options.c	2010-08-22 12:38:34.000000000 +0200
+++ ogmrip-0.13.6/libogmrip-gtk/ogmrip-x264-options.c	2010-11-22 19:30:10.484062522 +0100
@@ -246,7 +246,6 @@
       NULL);
   gtk_window_set_title (GTK_WINDOW (dialog), _("X264 Options"));
   gtk_window_set_resizable (GTK_WINDOW (dialog), FALSE);
-  gtk_dialog_set_has_separator (GTK_DIALOG (dialog), FALSE);
   gtk_container_set_border_width (GTK_CONTAINER (dialog), 5);
   gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_CLOSE);
   gtk_window_set_icon_from_stock (GTK_WINDOW (dialog), GTK_STOCK_PREFERENCES);
diff -u -r ogmrip-0.13.6.orig/libogmrip-gtk/ogmrip-xvid-options.c ogmrip-0.13.6/libogmrip-gtk/ogmrip-xvid-options.c
--- ogmrip-0.13.6.orig/libogmrip-gtk/ogmrip-xvid-options.c	2010-08-22 12:38:34.000000000 +0200
+++ ogmrip-0.13.6/libogmrip-gtk/ogmrip-xvid-options.c	2010-11-22 19:30:17.687023102 +0100
@@ -165,7 +165,6 @@
     return;
   }
 
-  gtk_dialog_set_has_separator (GTK_DIALOG (dialog), FALSE);
   gtk_dialog_add_buttons (GTK_DIALOG (dialog), GTK_STOCK_CLOSE, GTK_RESPONSE_CLOSE, NULL);
 
   gtk_window_set_title (GTK_WINDOW (dialog), _("XviD Options"));
diff -u -r ogmrip-0.13.6.orig/src/ogmrip-pref-dialog.c ogmrip-0.13.6/src/ogmrip-pref-dialog.c
--- ogmrip-0.13.6.orig/src/ogmrip-pref-dialog.c	2010-08-22 12:38:28.000000000 +0200
+++ ogmrip-0.13.6/src/ogmrip-pref-dialog.c	2010-11-22 19:30:32.120944110 +0100
@@ -211,7 +211,6 @@
   gtk_dialog_add_buttons (GTK_DIALOG (dialog),
       GTK_STOCK_CLOSE, GTK_RESPONSE_CLOSE,
       NULL);
-  gtk_dialog_set_has_separator (GTK_DIALOG (dialog), FALSE);
   gtk_container_set_border_width (GTK_CONTAINER (dialog), 5);
   gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_CLOSE);
   gtk_window_set_resizable (GTK_WINDOW (dialog), FALSE);
diff -u -r ogmrip-0.13.6.orig/src/ogmrip-profile-editor.c ogmrip-0.13.6/src/ogmrip-profile-editor.c
--- ogmrip-0.13.6.orig/src/ogmrip-profile-editor.c	2010-08-22 12:38:28.000000000 +0200
+++ ogmrip-0.13.6/src/ogmrip-profile-editor.c	2010-11-22 19:31:05.476761560 +0100
@@ -754,7 +754,6 @@
       GTK_STOCK_CLOSE, GTK_RESPONSE_CLOSE,
       NULL);
   gtk_window_set_resizable (GTK_WINDOW (dialog), FALSE);
-  gtk_dialog_set_has_separator (GTK_DIALOG (dialog), FALSE);
   gtk_container_set_border_width (GTK_CONTAINER (dialog), 5);
   gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_CLOSE);
   gtk_window_set_icon_from_stock (GTK_WINDOW (dialog), GTK_STOCK_PREFERENCES);
diff -u -r ogmrip-0.13.6.orig/src/ogmrip-profiles-dialog.c ogmrip-0.13.6/src/ogmrip-profiles-dialog.c
--- ogmrip-0.13.6.orig/src/ogmrip-profiles-dialog.c	2010-08-22 12:38:28.000000000 +0200
+++ ogmrip-0.13.6/src/ogmrip-profiles-dialog.c	2010-11-22 19:30:39.920901421 +0100
@@ -667,7 +667,6 @@
   gtk_dialog_add_buttons (GTK_DIALOG (dialog),
       GTK_STOCK_CLOSE, GTK_RESPONSE_CLOSE,
       NULL);
-  gtk_dialog_set_has_separator (GTK_DIALOG (dialog), FALSE);
   gtk_container_set_border_width (GTK_CONTAINER (dialog), 5);
   gtk_window_set_default_size (GTK_WINDOW (dialog), 450, -1);
   gtk_window_set_title (GTK_WINDOW (dialog), _("Edit Profiles"));
diff -u -r ogmrip-0.13.6.orig/src/ogmrip-progress-dialog.c ogmrip-0.13.6/src/ogmrip-progress-dialog.c
--- ogmrip-0.13.6.orig/src/ogmrip-progress-dialog.c	2010-08-22 12:38:28.000000000 +0200
+++ ogmrip-0.13.6/src/ogmrip-progress-dialog.c	2010-11-22 19:31:42.223560450 +0100
@@ -404,7 +404,6 @@
   gtk_window_set_default_size (GTK_WINDOW (dialog), 450, -1);
   gtk_window_set_icon_from_stock (GTK_WINDOW (dialog), GTK_STOCK_EXECUTE);
   gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_CANCEL);
-  gtk_dialog_set_has_separator (GTK_DIALOG (dialog), FALSE);
 
   area = gtk_dialog_get_content_area (GTK_DIALOG (dialog));
 
diff -u -r ogmrip-0.13.6.orig/src/ogmrip-queue-dialog.c ogmrip-0.13.6/src/ogmrip-queue-dialog.c
--- ogmrip-0.13.6.orig/src/ogmrip-queue-dialog.c	2010-08-22 12:38:28.000000000 +0200
+++ ogmrip-0.13.6/src/ogmrip-queue-dialog.c	2010-11-22 19:30:50.742842196 +0100
@@ -611,7 +611,6 @@
 
   gtk_window_set_title (GTK_WINDOW (dialog), _("Encoding Queue"));
   gtk_window_set_default_size (GTK_WINDOW (dialog), 500, 300);
-  gtk_dialog_set_has_separator (GTK_DIALOG (dialog), FALSE);
   gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_ACCEPT);
   gtk_dialog_set_response_sensitive (GTK_DIALOG (dialog), GTK_RESPONSE_ACCEPT, FALSE);
   gtk_window_set_icon_from_stock (GTK_WINDOW (dialog), GTK_STOCK_PROPERTIES);
diff -u -r ogmrip-0.13.6.orig/src/ogmrip-update-dialog.c ogmrip-0.13.6/src/ogmrip-update-dialog.c
--- ogmrip-0.13.6.orig/src/ogmrip-update-dialog.c	2010-08-22 12:38:28.000000000 +0200
+++ ogmrip-0.13.6/src/ogmrip-update-dialog.c	2010-11-22 19:31:14.786710608 +0100
@@ -130,7 +130,6 @@
       GTK_STOCK_CANCEL, GTK_RESPONSE_REJECT,
       GTK_STOCK_OK, GTK_RESPONSE_ACCEPT,
       NULL);
-  gtk_dialog_set_has_separator (GTK_DIALOG (dialog), FALSE);
   gtk_container_set_border_width (GTK_CONTAINER (dialog), 5);
   gtk_window_set_default_size (GTK_WINDOW (dialog), 450, 350);
   gtk_window_set_title (GTK_WINDOW (dialog), _("Update profiles"));
