Index: src/ogmrip-profile-editor.c
===================================================================
--- src/ogmrip-profile-editor.c	(révision 2035)
+++ src/ogmrip-profile-editor.c	(copie de travail)
@@ -951,7 +951,7 @@
   dialog->priv->audio_quality_label = glade_xml_get_widget (xml, "audio-quality-label");
   dialog->priv->audio_quality_spin = glade_xml_get_widget (xml, "audio-quality-spin");
   ogmrip_settings_bind (settings, dialog->priv->profile_section,
-      OGMRIP_GCONF_AUDIO_QUALITY, G_OBJECT (widget), "value");
+      OGMRIP_GCONF_AUDIO_QUALITY, G_OBJECT (dialog->priv->audio_quality_spin), "value");
 
   widget = glade_xml_get_widget (xml, "normalize-check");
   ogmrip_settings_bind (settings, dialog->priv->profile_section,
